### DISCUSSION TITLE
Rocky Linux CI

### DISCUSSION THREAD

**Discussion by mzuranski** [Unidata/gempak#107] [Q&A]
As of now we run CI actions for Ubuntu and CentOS 6 & 7.  CentOS 7 has some life in it yet but 6 is already EOL. https://endoflife.software/operating-systems/linux/centos

[Rocky Linux is more or less the successor to CentOS](https://rockylinux.org/about), and [Unidata has already adopted it for some of their operations](https://www.unidata.ucar.edu/software/mcidas/adde_servers.html).  I'm admittedly not familiar with GitHub Actions or CI/CD, but I'm wondering if that's something we could setup for Rocky?

**Comment by akrherz** [comment]
Agreed, presently the centos images are coming via Unidata's docker repo with some things pre-setup that are not fully testing GEMPAK builds in CI :(

**Reply by mzuranski** [reply]
I'm guessing that's a reference to [this comment](https://github.com/Unidata/gempak/pull/106#issuecomment-1168628584) from earlier.

Pardon my ignorance here, but does GitHub offer a number of different distros and CentOS just isn't one of them?  I tried looking into this earlier and it seemed like Ubuntu is and then like you said you had to setup CentOS a different way.

**Reply by akrherz** [reply]
See [supported runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)

**Reply by mzuranski** [reply]
Thanks for linking that.  I came across it earlier but going over it again (and falling down the rabbit hole a ways) things are starting to click better now.

So are you thinking that starting from a different docker container might address the issues there?  If so I just came across this, looks like it was recently added: https://hub.docker.com/r/unidata/rockylinux  Given the lack of overview though no idea what to expect.
